src/libraries/Microsoft.PowerFx.LanguageServerProtocol/JsonRpc/JsonRpcHelper.cs

```diff
@@ -25,7 +25,7 @@ public enum ErrorCode
             InvalidRequest = -32600,
             MethodNotFound = -32601,
             InvalidParams = -32602,
-            InternalError = -30603,
+            InternalError = -32603,
```

Comment:
This looks like a typo in the original. 

See: 
https://microsoft.github.io/language-server-protocol/specifications/specification-3-14/